apps/docs/content/components/OrderedList/index.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: Ordered list
+description: Use the ordered list component to display a list of numbered items.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Anatomy
+
+![An image showing several sentences as a linear list of numbered items](https://github.com/primer/brand/assets/912236/5cda0c94-c120-4fc6-8306-8b55012dee0a)
+
+## Usage
+
+Use an ordered list to represent a list of items in a numerical order.
+
+A number indicating the items relative position in the sequence will be prepended to each item in the list.
+
+The ordered list is commonly used when presenting sequential or step-by-step information. It is suitable for instructions, procedures, tutorials, or any content that follows a specific order. Ensure that the items in the ordered list are logically organized and follow a clear progression. This helps readers understand the sequence of information and makes it easier to follow.
+
+Avoid overusing ordered lists. Reserve them for situations where the order of items is genuinely meaningful or where a clear sequence needs to be conveyed. For
+lists that don't require a specific order, consider using an [unordered list](/components/UnorderedList) instead.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/40022d10-248f-45d5-acce-2917a72d7b12"
+      alt=""
+    />
+    <Caption>Use an ordered list to present a sequence of steps.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/e9dd1ee2-cfa7-40d4-a8b6-2416cf99cdae"
+      alt=""
+    />
+    <Caption>
+      Don't use an ordered list to present a list of items that don't require a
+      specific order. Use an unordered list instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+The ordered list should have concise and short items. Avoid using long sentences or paragraphs as they can be difficult to scan and comprehend. If you need to include a long item or even headings and links, consider using a [pillar](/components/Pillar), a [card](/components/Card) or a [river](/components/River) instead.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://github.com/primer/brand/assets/912236/181f1cb9-05bc-4010-a80b-a978b7ffbe56"
+      alt=""
+    />
+    <Caption>Use an ordered list for short and concise content items.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://github.com/primer/brand/assets/912236/28950f6a-d7f0-4d9d-bb70-02c062d309f3"
+      alt=""
+    />
+    <Caption>
+      Don't use an ordered list for long and rich sentences, paragraphs,
+      headings or links. Use a pillar, card or river components instead.
+    </Caption>
+  </Dont>
+</DoDontContainer>
+
+## Options
```

Comment:
This is empty, do you want to add something or actually just remove the heading?